Reference/google/protobuf/any_test.pb.swift

```diff
@@ -107,6 +107,7 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Proto3Message, SwiftProtobuf._Mes
   init() {}
 
   mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
+    _ = _uniqueStorage()
```

Comment:
FYI:  I've been toying with the idea of changing `_uniqueStorage()` to not return the reference.  It's not 1.0 critical since it's both defined and used purely within generated code (so it's not a library API issue).